tools/c7n_mailer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-alpine
+
+RUN apk add --no-cache git
```

Comment:
are gcc and musl-dev not required as well? last time I set up a mailer docker image I had to install those atop the base image